terraform/SLI_3_days_views.tf

```diff
@@ -1,6 +1,6 @@
 resource "google_bigquery_table" "census_data_3_days_ago_view" {
```

Comment:
We could create a module for those X days SLI :)

Reply:
Decide offline to skip it